cl/settings/third_party/elasticsearch.py

```diff
@@ -4,9 +4,10 @@
 
 ELASTICSEARCH_DISABLED = env(
```

Comment:
Should we just remove this setting completely?

Reply:
Never mind, I remembered this will morph into a setting for PA, and another for OA, and C, and so forth...

Reply:
Yes, exactly. On OA, I'll change the name of this setting so that we can reuse it to control the search engine transition.